optimize/backend/src/it/java/io/camunda/optimize/service/importing/processdefinition/ZeebeProcessDefinitionImportIT.java

```diff
@@ -68,7 +68,11 @@ public void importZeebeProcess_allDataSavedToDefinition() {
               assertThat(importedDef.getKey()).isEqualTo(deployedProcess.getBpmnProcessId());
               assertThat(importedDef.getVersion())
                   .isEqualTo(String.valueOf(deployedProcess.getVersion()));
-              assertThat(importedDef.getVersionTag()).isEqualTo(ZeebeBpmnModels.VERSION_TAG);
+              if (isZeebeVersionPre86()) {
+                assertThat(importedDef.getVersionTag()).isNull();
```

Comment:
I'm ok with this solution. Will the BPMN be deployable though on older compatibility tests given that it contains version tags?

Reply:
You mean the deployment that happens in the `given` (l55)? The tests only failed at the assertion so seems to be fine deploying.